### ISSUE TITLE
[Blank Pages]: O3DE User Guide

### ISSUE THREAD

**Issue by LB-RobertBaczkowski** [o3de/o3de.org#240]
The following pages are completely empty and do not have neither placeholder information nor a preview window.

- [ ] https://o3deorg.netlify.app/docs/user-guide/visualization/cameras/
- [ ] https://o3deorg.netlify.app/docs/user-guide/visualization/cameras/setup-and-rigging/
- [ ] https://o3deorg.netlify.app/docs/user-guide/visualization/environments/
- [x] https://o3deorg.netlify.app/docs/user-guide/visualization/rendering/
- [x] https://o3deorg.netlify.app/docs/user-guide/visualization/rendering/atom-in-o3de/
- [x] https://o3deorg.netlify.app/docs/user-guide/visualization/rendering/shaders/

**Comment by chanmosq** [comment]
This link has been removed during initial documentation development: https://o3deorg.netlify.app/docs/user-guide/visualization/rendering/atom-in-o3de/

**Comment by LB-RobertBaczkowski** [comment]
The links to empty pages on https://o3de.org/docs/user-guide/visualization/ have been removed as described. However, the first link ("Animation Editor") redirects to an empty page called "Animation" (https://o3de.org/docs/user-guide/visualization/animation/ ). This page should be remade like the corresponding page for "Cinematics" (https://o3de.org/docs/user-guide/visualization/cinematics/), to include further links.

**Comment by chanmosq** [comment]
Hey @LB-RobertBaczkowski, I checked off the other links you listed in the issue description. I verified that those pages were marked as `draft: true` so they don't show up on the website anymore, and no other active pages reference them. 

It seems the issue you're describing is covered by the open issue, **[Blank Pages]: O3DE User Guide: Animation, Animation Editor, Character Editor** (https://github.com/o3de/o3de.org/issues/118). We'll continue tracking the problem there.
